action.yml

```
@@ -0,0 +1,46 @@
+name: 'Provision GitHub Runners'
+description: 'Scale up ECS Service count and deploy a task definition'
+inputs:
+  aws-access-key-id:
+    description: >-
+      AWS Access Key ID. This input is required if running in the GitHub hosted environment.
+      It is optional if running in a self-hosted environment that already has AWS credentials,
+      for example on an EC2 instance.
+    required: false
+  aws-secret-access-key:
+    description: >-
+      AWS Secret Access Key. This input is required if running in the GitHub hosted environment.
+      It is optional if running in a self-hosted environment that already has AWS credentials,
+      for example on an EC2 instance.
+    required: false
+  aws-region:
+    description: 'Your AWS region'
+    required: true
+    default: 'us-east-1'
+  ecr_repository:
```

mix of - and _ in the input names. Recommend just using - 
```suggestion
  ecr-repository:
```